tutorials/thermal_camera.md

```diff
@@ -299,7 +299,7 @@ Here's an example 16-bit thermal camera subscriber that performs this conversion
 ```c
 #include <cstdint>
 
-#include <gz/msgs.hh>
+#include <gz/msgs/image.pb>
```

Comment:
`<gz/msgs/image.pb.h>`?

Reply:
Instead of including all messages, we just include the one that is required fot this example

Reply:
oh I mean the `.h` fix extension is missing

Reply:
ups https://github.com/gazebosim/gz-sensors/pull/270/commits/ed35f8243c1c3ca70a90932b75755a8057ba72a6